2021-10-19 16:48:40,329 INFO    MainThread:32131 [wandb_setup.py:_flush():69] setting env: {}
2021-10-19 16:48:40,329 INFO    MainThread:32131 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-19 16:48:40,329 INFO    MainThread:32131 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '8bbaf831241b8525faf8ba0890cc99d295161186'}
2021-10-19 16:48:40,329 INFO    MainThread:32131 [wandb_setup.py:_flush():69] setting login settings: {}
2021-10-19 16:48:40,329 INFO    MainThread:32131 [wandb_init.py:_log_setup():342] Logging user logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211019_164840-1dowej0f/logs/debug.log
2021-10-19 16:48:40,329 INFO    MainThread:32131 [wandb_init.py:_log_setup():343] Logging internal logs to /root/konst/ml-utils/training/ner_employment/wandb/run-20211019_164840-1dowej0f/logs/debug-internal.log
2021-10-19 16:48:40,330 INFO    MainThread:32131 [wandb_init.py:init():375] calling init triggers
2021-10-19 16:48:40,330 INFO    MainThread:32131 [wandb_init.py:init():382] wandb.init called with sweep_config: {}
config: {'data_path': '/root/konst/data/bio_dataset/bio_ner_dataset_fam.csv', 'checkpoint_path': '/root/konst/model_checkpoints/ner_employment', 'model_path': 'ner_base_epoch_8_acc_0.4811_vs_0.0755_loss_-11.2523_vs_-11.2842.pth', 'batch_size': 128, 'epochs': 100, 'start_epoch': 0, 'test_size': 0.05, 'lam': 0, 'patience': 5, 'factor': 0.5, 'use_wandb': True, 'lr': 0.001, 'save_at': 1, 'adjust_lr_every': 1, 'num_workers': 4}
2021-10-19 16:48:40,330 INFO    MainThread:32131 [wandb_init.py:init():424] starting backend
2021-10-19 16:48:40,330 INFO    MainThread:32131 [backend.py:_multiprocessing_setup():72] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-10-19 16:48:40,335 INFO    MainThread:32131 [backend.py:ensure_launched():135] starting backend process...
2021-10-19 16:48:40,339 INFO    MainThread:32131 [backend.py:ensure_launched():140] started backend process with pid: 32146
2021-10-19 16:48:40,340 INFO    MainThread:32131 [wandb_init.py:init():429] backend started and connected
2021-10-19 16:48:40,342 INFO    MainThread:32131 [wandb_init.py:init():477] updated telemetry
2021-10-19 16:48:40,343 INFO    MainThread:32131 [wandb_init.py:init():500] communicating current version
2021-10-19 16:48:40,829 INFO    MainThread:32131 [wandb_init.py:init():505] got version response upgrade_message: "wandb version 0.12.4 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2021-10-19 16:48:40,829 INFO    MainThread:32131 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-10-19 16:48:41,018 INFO    MainThread:32131 [wandb_init.py:init():540] starting run threads in backend
2021-10-19 16:48:41,793 INFO    MainThread:32131 [wandb_run.py:_console_start():1584] atexit reg
2021-10-19 16:48:41,794 INFO    MainThread:32131 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.REDIRECT
2021-10-19 16:48:41,795 INFO    MainThread:32131 [wandb_run.py:_redirect():1463] Redirecting console.
2021-10-19 16:48:41,796 INFO    MainThread:32131 [wandb_run.py:_redirect():1519] Redirects installed.
2021-10-19 16:48:41,796 INFO    MainThread:32131 [wandb_init.py:init():565] run started, returning control to user process
2021-10-19 16:51:13,650 INFO    MainThread:32131 [wandb_run.py:_atexit_cleanup():1554] got exitcode: 255
2021-10-19 16:51:13,651 INFO    MainThread:32131 [wandb_run.py:_restore():1526] restore
2021-10-19 16:51:15,873 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 85452
  total_bytes: 85452
}

2021-10-19 16:51:16,092 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 1
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 85452
  total_bytes: 85452
}

2021-10-19 16:51:17,075 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 85452
  total_bytes: 93481
}

2021-10-19 16:51:17,178 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 85452
  total_bytes: 93481
}

2021-10-19 16:51:17,279 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 93481
  total_bytes: 93481
}

2021-10-19 16:51:17,381 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 93481
  total_bytes: 93481
}

2021-10-19 16:51:17,484 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 93481
  total_bytes: 93481
}

2021-10-19 16:51:17,587 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 93481
  total_bytes: 93481
}

2021-10-19 16:51:17,689 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 93481
  total_bytes: 93481
}

2021-10-19 16:51:17,791 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 93481
  total_bytes: 93481
}

2021-10-19 16:51:17,894 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 93481
  total_bytes: 93481
}

2021-10-19 16:51:18,220 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 93481
  total_bytes: 93481
}

2021-10-19 16:51:18,323 INFO    MainThread:32131 [wandb_run.py:_wait_for_finish():1676] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
  other_count: 5
}
pusher_stats {
  uploaded_bytes: 93481
  total_bytes: 93481
}

2021-10-19 16:51:19,509 INFO    MainThread:32131 [wandb_run.py:_show_files():1898] logging synced files
